src/NuGet.Core/NuGet.Common/CommandLineArgumentCombinationException.cs

```diff
@@ -5,6 +5,7 @@
 
 namespace NuGet.Common
 {
+    [SerializableAttribute]
     public class CommandLineArgumentCombinationException : Exception, ILogMessageException
     {
         private readonly ILogMessage _logMessage;
```

Comment:
This class needs to implement `ISeriablizable` properly for `_logMessage` to round trip.